src/applications/hca/validation.js

```diff
@@ -79,10 +77,11 @@ export function validateDependentDate(
   messages,
   index,
 ) {
-  const dependent = moment(dependentDate);
-  const dob = moment(_.get(`dependents[${index}].dateOfBirth`, formData));
+  const dependent = Date.parse(dependentDate);
```

Comment:
`dependentDate` is the second argument in `validateDependentDate`, and it is also an ISO date string.

https://github.com/department-of-veterans-affairs/vets-website/blob/8f57a94c2e86cc81b6d388d0af41c64af6286184/src/applications/hca/tests/validation.unit.spec.js#L131-L145